codenames_backend/routes/games.py

```diff
@@ -25,6 +29,92 @@ def new_game():
     db.session.commit()
     return game_schema.jsonify(game), 201
 
+def get_smallest_team(game_id):
+    teams = (
```

Comment:
this could be done more concisely with:

```python
blue_team_size = Player.query.filter_by(team="blue", game_id=game_id).count()
red_team_size = Player.query.filter_by(team="red", game_id=game_id).count()
return "blue" if blue_team_size < red_team_size else "red"
```

Reply:
Yup. That is simpler. My lack of python etl experience is showing. Fix applied.